### PR TITLE
fix: Add function handling for relativeTime.future and relativeTime.past

### PR DESCRIPTION
**Pull request by drgx**
Fix: #1196

### CODE REVIEW

**Comment by drgx** [comment]
Will fix the test

**Comment by iamkun** [comment]
Thanks, please add the relevant unit test to make the CI pass.

**Comment by drgx** [comment]
Updated please kindly help to review 🙇

File: src/locale/lv.js [hunk 1]
```diff
@@ -17,6 +17,21 @@ const locale = {
     LL: 'YYYY. [gada] D. MMMM',
     LLL: 'YYYY. [gada] D. MMMM, HH:mm',
     LLLL: 'YYYY. [gada] D. MMMM, dddd, HH:mm'
+  },
+  relativeTime: {
```
Comment (iamkun) [inline src/locale/lv.js:21]: is this a merge error?

Comment (drgx) [reply src/locale/lv.js:21]: let me rebase to latest master

Comment (drgx) [reply src/locale/lv.js]: oops my bad.. fixed..

**Comment by iamkun** [comment]
LGTM

**Comment by skywritergr** [comment]
I actually had the same issue. Is there a plan to release this in the near future? Thanks for all the hard work.

**Comment by iamkun** [comment]
@skywritergr will do it in the next week I hope, before I could fix another bug.

**Pull request merged**